### ISSUE TITLE
Just a quick reminder. 

### ISSUE THREAD

**Issue by mysr3809** [sahandghavidel/mern-estate#6]
After upload an image then if we have an error, we do not set false setFileUploadError value so when i try to upload new image i saw the error always. To fix that  we should set  setFileUploadError value to the false and the process can start from the beginning.

I appreciate for your effort @sahandghavidel 

`
const handleFileUpload = (file) => {

    setFileUploadError(false);

    const storage = getStorage(app);

    const fileName = new Date().getTime() + file.name;

    const storageRef = ref(storage, fileName);

    const uploadTask = uploadBytesResumable(storageRef, file);`

**Comment by Atteq786** [comment]
You're right! Resetting setFileUploadError(false) at the start of the upload process ensures we clear any previous error states and allow a fresh attempt.
